src/app/GameController.ts

```diff
@@ -0,0 +1,70 @@
+import { time } from 'console';
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { Player } from '../Player';
+import { BreedProcessor } from './BreedProcessor';
+import { Timer } from './Timer';
+import { View } from './View';
+
+export class GameController {
+  private currentPlayer: Player | undefined;
+  private player: Player | undefined;
+  private timer: Timer;
+  private breedProcessor: BreedProcessor;
+  constructor(private view: View) {
+    this.timer = new Timer();
+    this.breedProcessor = new BreedProcessor();
+  }
+
+  get theCurrentPlayer(): Player {
+    return this.currentPlayer as Player;
+  }
+
+  get theTimer(): Timer {
+    return this.timer;
+  }
+
+  initializePlayer(name: string, imgPath: string): Player {
+    this.player = new Player(name, imgPath);
+    this.currentPlayer = this.player;
```

Comment:
currentPlayer will be important when we will have multiple players, is that it?

Reply:
yes, it will be slightly refactored, and current player will be mainly rendered